src/main/kotlin/ca/kittle/plugins/Databases.kt

```diff
@@ -39,9 +42,20 @@ fun Application.connectToMongoDB(): MongoDatabase {
     val databaseName = environment.config.tryGetString("db.mongo.database.name") ?: "PartyConnections"
 
     val credentials = user.let { userVal -> password?.let { passwordVal -> "$userVal:$passwordVal@" } }.orEmpty()
-    val uri = "mongodb://$credentials$host:$port/?replicaSet=rs0&maxPoolSize=$maxPoolSize&w=majority"
+    val uri = "mongodb://$credentials$host:$port/?replicaSet=rs0&maxPoolSize=$maxPoolSize&w=majority&connectTimeoutMS=1500"
 
-    val mongoClient = MongoClients.create(uri)
+//    val mongoClient = MongoClients.create(uri)
+    val mongoClient = MongoClients.create(
```

Comment:
**suggestion (performance):** Consider adjusting connection timeout values

The current timeout values (1 second for connection, 15 seconds for read) might need adjustment based on your specific network conditions and database response times. Consider profiling your application to determine more appropriate values.